hydra-redis/hydra_graph.py

```diff
@@ -64,26 +66,32 @@ def get_endpoints(api_doc):
     redis_graph.add_node(entrypoint_node)
     return get_apistructure(entrypoint_node, api_doc)
 
-
-if __name__ == "__main__":
+def main(new_url,api_doc):
     redis_con = redis.Redis(host='localhost', port=6379)
+    global redis_graph
```

Comment:
Instead of creating a global variable for the graph you should pass the graph as a function argument whenever required. **Global variables should be used very carefully. You should avoid using them unless it's absolutely necessary.**

Reply:
Actually, we need to import redis_graph attribute in querying_mechanism. So, I am using this as a global variable.